tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1239,9 +1239,9 @@ a dictionary can also be used with a named enum.
         # Use constant dictionary functions
         # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
         print(State.keys())
-        # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
+        # prints '{ "STATE_IDLE": 0, "STATE_JUMP": 5, "STATE_SHOOT": 7 }'
         print(State)
-        # prints '[0, 5, 6]'
+        # prints '[0, 5, 7]'
```

Comment:
The order is reversed, `print(State)` prints a dictionary, and `print(State.keys())` prints an array of keys (here the values).

Reply:
That's `print(State.values())`, it's above the line that has the command

Reply:
It's:
```gdscript
print(State.keys()) # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
print(State)  # prints '{ "STATE_IDLE": 0, "STATE_JUMP": 5, "STATE_SHOOT": 7 }'
print(State.values()) # prints '[0, 5, 7]'
```

Reply:
I would like to add spaces between so it's more clear, but it seems every little change I add ruins something else, so I will commit only the suggestions :+1: